clustermesh/clustermesh.go

```diff
@@ -905,6 +905,16 @@ func (k *K8sClusterMesh) Connect(ctx context.Context) error {
 			aiLocal.ClusterName, aiLocal.ClusterID)
 	}
 
+	cid, err := strconv.Atoi(aiRemote.ClusterID)
+	if err != nil {
+		k.Log("❌ Cluster has the non-numeric ID 0 (cluster ID: %s): %s", aiLocal.ClusterID, err)
+		return err
+	}
+	if cid > 255 || cid < 1 {
+		return fmt.Errorf("local cluster has the cluster ID out of acceptable range (e.g. 1-255) (cluster ID: %d)",
```

Comment:
Isn't this the remote cluster ID?

```suggestion
		return fmt.Errorf("remote cluster has the cluster ID out of acceptable range (e.g. 1-255) (cluster ID: %d)",
```